apps/docs/src/development/theming.md

```diff
@@ -0,0 +1,20 @@
+# Theming Setup
+
+Onyx supports a dark and a light theme.  
+To learn more about the theming concept of onyx, take a look at our [colors documentation](/basics/colors.html)
+
+In order to let the user choose the preferred theme, we recommend [useDark of vueuse](https://vueuse.org/core/useDark/#usedark).  
+Combine it as follows with an onyx button to let the users set the theme of their choice:
```

Comment:
Jannicks intended way is to have an option in the OnyxUserMenu inside the nav bar so we should recommend to use a button here

Reply:
https://www.figma.com/design/YfEUBOHk4J4nYrk04geswG/onyx---Design-System-Figma-Library?node-id=1709-12347&m=dev

Reply:
we don't support that feature yet, do we?

Reply:
No not yet :/ Should we check with Jannick how the handle dark mode toggle until we have this? Maybe as a switch/icon button in the context area of the nav bar

Reply:
ok did it like that for now ^^